test/e2e/packagemanifest_e2e_test.go

```diff
@@ -85,6 +85,9 @@ var _ = Describe("Package Manifest", func() {
 		require.NotNil(GinkgoT(), pm)
 		require.Equal(GinkgoT(), packageName, pm.GetName())
 		require.Equal(GinkgoT(), expectedStatus, pm.Status)
+		require.Equal(GinkgoT(), "0.0.0", pm.Status.Channels[0].CurrentCSVDesc.MinKubeVersion)
+		require.Equal(GinkgoT(), *dummyImage, pm.Status.Channels[0].CurrentCSVDesc.RelatedImages[0])
+		require.Equal(GinkgoT(), csv.Spec.NativeAPIs, pm.Status.Channels[0].CurrentCSVDesc.NativeAPIs)
 		require.Equal(GinkgoT(), "label", pm.GetLabels()["projected"])
```

Comment:
It would be great if we could add a test that makes sure that if a bundle has a csv with `spec.relatedImages` field that those are also included